### ISSUE TITLE
Java Documentation comments

### ISSUE THREAD

**Issue by bhaskar-yp** [duffelhq/duffel-api-java#65]
Can we use Java documentation comments (/** ... */) instead of 
```
///  <summary>
///  A machine-readable identifier for this specific error
///  </summary>
```
So the information is available in the IDE and also for doc generation tools.

**Comment by johnpeterharvey** [comment]
As far as I can see, I fixed those in version 1.0.1, back in September 2022. Could you please confirm which version you're seeing this in? Thanks!

**Comment by johnpeterharvey** [comment]
Fixed as far as I can see.
